.eslintrc.js

```diff
@@ -1,19 +1,20 @@
 module.exports = {
   env: {
-    commonjs: true,
-    es6: true,
+    es2021: true,
     node: true
   },
   extends: [
     'standard'
   ],
-  globals: {
-    Atomics: 'readonly',
-    SharedArrayBuffer: 'readonly'
-  },
+  parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 2018
+    ecmaVersion: 12,
+    sourceType: 'module'
   },
+  plugins: [
+    '@typescript-eslint'
+  ],
   rules: {
+    'no-undef': 'off'
```

Comment:
Why are we turning off the no-undef rule? Seems like a pretty big omission 🤔

Reply:
TSC catches undef errors, eslint has issues with things like interfaces being "undefined"